webapp/app/dataExport/utils/format.js

```diff
@@ -123,3 +131,15 @@ export const getCustomVariableI18nMappings = (i18nKey) => {
   const key = i18nKey.split('.').pop()
   return variableI18nMappings[key] ? variableI18nMappings[key] : i18nKey
 }
+
+const unitI18nMappings = {
+  ha: 'ha',
+  kmSq: 'kmSq',
+  mileSq: 'mileSq',
+  acre1000: 'acre1000',
+  acre: 'acre',
+  haMillion: 'haMillion',
+  [UnitSpec.units.haThousand]: UnitSpec.units.haThousand,
+}
```

Comment:
are all units included ?

Reply:
Contains i18n for units in conversion, other units are under UnitSpec